app/uk/gov/hmrc/gform/sharedmodel/email/LocalisedEmailTemplateId.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.sharedmodel.email
+
+import play.api.libs.json.{ JsError, JsObject, JsString, JsSuccess, Reads }
+import uk.gov.hmrc.gform.sharedmodel.{ EmailVerifierService, LangADT, LocalisedString }
+import uk.gov.hmrc.gform.sharedmodel.notifier.NotifierTemplateId
+
+case class LocalisedEmailTemplateId(emailTemplateId: String, maybeEmailTemplateIdCy: Option[String]) {
```

Comment:
LocalisedEmailTemplateId seems to be used in `UploadableEmailDestination` which used when uploading template in gform backend. Do we need this class and all the dependencies that come with it?

Reply:
No, we don't need them, but it's rabbit hole when I tried to remove them. We should create a new ticket for that.